Makefile

```diff
@@ -44,6 +45,7 @@ glideup:
 build:
 	${DEV_ENV_PREFIX} -e CGO_ENABLED=0 ${DEV_ENV_IMAGE} go build -a -installsuffix cgo -ldflags ${LDFLAGS} -o ${BINARY_DEST_DIR}/boot boot.go || exit 1
 	@$(call check-static-binary,$(BINARY_DEST_DIR)/boot)
+	goupx ${BINARY_DEST_DIR}/boot
```

Comment:
I'd personally wait for that to be merged to minimize this PR. It won't require users to install `upx` on their machines in order for this to work. :)